tools/data-model-exporter/data_model_exporter/tests/fixtures/test.ttl

```diff
@@ -0,0 +1,89 @@
+# baseURI: https://zombo.com/zombo
+# imports: http://www.w3.org/ns/dcat
+# prefix: zombo
+
+@prefix : <https://zombo.com/zombo#> .
+@prefix zombo: <https://zombo.com/zombo#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix duo: <http://purl.obolibrary.org/obo/duo-basic.owl#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
```

Comment:
can probably nix a bunch of these if we need to, a lot are unused

Reply:
Hmm. Not sure.  These are standard and may be pulled in behind the scenes -- in other imported terms.  I'd keep them for now.

Reply:
this is just the fake TTL schema i'm using to test that the generated JSON schema looks how we expect based on a controlled input, for what it's worth